src/main/java/org/assertj/core/api/Array2DAssert.java

```diff
@@ -79,20 +79,20 @@
    * <p>
```

Comment:
`Verifies that the number of values in the actual array is equal to the given one.`
should be reworded to:
`Verifies that the actual 2D array has the same dimensions as the given one.`

Reply:
done